src/components/NavigationBar.jsx

```diff
@@ -10,8 +14,18 @@ export default function NavigationBar() {
 	const changeToFalse = useStore(state => state.changeToFalse);
 	const timerOn = useStore(state => state.timerOn);
 
+	const [isShown, setIsShown] = useState(false);
+
+	function handleVisibility() {
+		setIsShown(!isShown);
+	}
+
 	return (
 		<StyledNavigationBar>
+			<StyledButton type="button" onClick={handleVisibility}>
+				<SaveIcon></SaveIcon>
+			</StyledButton>
+			{isShown ? <SaveModalForm onCancel={handleVisibility} /> : ''}
```

Comment:
Try to use the && operator 
=> 
```javascript
{isShown && <SaveModalForm onCancel={handleVisibility} />}
```

Then you don't need to use the else statement

Reply:
very nice thank you!